### PR TITLE
430 New Key Type Documentation

### PR DESCRIPTION
**Pull request by ACStoneCL**
### Related links

[Expand the Keys section to include new key types in the upcoming 1.5.0 release #430](https://github.com/casper-network/docs/issues/430)

### Changes

Adding `Unbond` and `ChainspecRegistry` to the various list of keys.

### Notes

Ran locally without issue.

### CODE REVIEW

File: source/docs/casper/design/serialization-standard.md [hunk 1]
```diff
@@ -485,6 +485,8 @@ A _key_ in the [Global State](./global-state.md#global-state-intro) is one of th
 -   32-byte Auction withdrawal identifier
 -   32-byte dictionary identifier
```
Comment (darthsiroftardis) [inline source/docs/casper/design/serialization-standard.md:486]:
```suggestion
-   32-byte Dictionary identifier
```

File: source/docs/casper/design/serialization-standard.md [hunk 2]
```diff
@@ -561,7 +567,9 @@ There are three types of actions that can be done on a value: read, write, add.
 | Bid      | System                  |
 | Withdraw | System                  |
 | Dictionary | Read (via API)        |
-| SystemContractRegistry | System     |
+| SystemContractRegistry | System    |
```
Comment (darthsiroftardis) [inline source/docs/casper/design/serialization-standard.md:570]:
```suggestion
| SystemContractRegistry | Read (via API)    |
```

Same for Chainspec registry

**Review by darthsiroftardis — COMMENTED** [review]
Seeing some files here that aren't necessarily part of this PR, i.e that of authorized keys and fast sync

**Comment by ipopescu** [comment]
> Seeing some files here that aren't necessarily part of this PR, i.e that of authorized keys and fast sync

Yes, I think we need to finish the rebase, on which Luca and Gloreen were working.

File: source/docs/casper/design/serialization-standard.md [hunk 3]
```diff
@@ -561,7 +567,9 @@ There are three types of actions that can be done on a value: read, write, add.
 | Bid      | System                  |
 | Withdraw | System                  |
 | Dictionary | Read (via API)        |
-| SystemContractRegistry | System     |
+| SystemContractRegistry | Read (via API)    |
+| Unbond   | System                  |
+| ChainspecRegistry | System         |
```
Comment (darthsiroftardis) [inline source/docs/casper/design/serialization-standard.md:572]:
```suggestion
| ChainspecRegistry | Read (via API)       |
```

**Review by darthsiroftardis — APPROVED** [review]
One last nit

**Pull request merged**